changelog/2311.fixed.md

```diff
@@ -0,0 +1 @@
+Fixed `::unistd::Group::members` crash on misaligned read by read_unaligned use
```

Comment:
This sentence is hard to understand.  Could you please try rewording it?

Reply:
Done